internal/providers/terraform/ibm/testdata/container_vpc_cluster_test/container_vpc_cluster_test.golden

```diff
@@ -3,10 +3,10 @@
                                                                                                                                        
  ibm_is_vpc.vpc1                                                                                                                       
  ├─ VPC instance                                                                                             1  Instance         $0.00 
- └─ VPC egress us-south 0 - 9995 GB                                                                      9,995  GB             $869.57 
- └─ VPC egress us-south 9995 - 49995 GB                                                                 40,000  GB           $3,320.00 
- └─ VPC egress us-south 49995 - 149995 GB                                                                  105  GB               $7.35 
- └─ VPC egress us-south 149995 - 99999999999999 GB                                                           0  GB               $0.00 
+ └─ VPC egress us-south (first 9995 GB)                                                                  9,995  GB             $869.57 
```

Comment:
I like the idea, but I think specifying the tiers as a range is better for two reasons: it matches with the way they are presented in the pricing pages and it minimizes the use of english words, since there isn't a way to offer translations for words like (next, first, over, etc..)

Reply:
I made to change to bring the tiers more in line with the currect infracost implementation.
For example:
```
 google_storage_bucket.EuMulti                                                                                                                
 ├─ Storage (standard)                                                                            150  GiB                              $3.90 
 ├─ Object adds, bucket/object list (class A)                                                       4  10k operations                   $0.20 
 ├─ Object gets, retrieve bucket/object metadata (class B)                                          2  10k operations                   $0.01 
 └─ Network egress                                                                                                                            
    ├─ Data transfer in same continent                                                            550  GB                               $5.50 
    ├─ Data transfer to worldwide excluding Asia, Australia (first 1TB)                         1,024  GB                             $122.88 
    ├─ Data transfer to worldwide excluding Asia, Australia (next 9TB)                          9,216  GB                           $1,013.76 
    ├─ Data transfer to worldwide excluding Asia, Australia (over 10TB)                         2,260  GB                             $180.80 
    ├─ Data transfer to Asia excluding China, but including Hong Kong (first 1TB)               1,024  GB                             $122.88 
    ├─ Data transfer to Asia excluding China, but including Hong Kong (next 9TB)                  476  GB                              $52.36 
    ├─ Data transfer to China excluding Hong Kong (first 1TB)                                      50  GB                              $11.50 
    └─ Data transfer to Australia (first 1TB)                                                     250  GB                              $47.50 
```